### ISSUE TITLE
Implement new certifier info flow (required email)

### ISSUE THREAD

**Issue by michaelclement** [department-of-veterans-affairs/va.gov-team#96234]
As the IVC team, we need to make sure that every user who fills out 10-10d is providing a valid email address so that we can contact them in the event of a silent failure.

[Figma](https://www.figma.com/design/UmAtr3ULQEInMXfNFwP0g0/Application-for-CHAMPVA?node-id=3998-26815&t=iP6DzDhzaRHGRUgb-4)

**Explanation**
[Based on this ticket](https://app.zenhub.com/workspaces/ivc-forms-652da2d3f0ae4c0016bfb109/issues/gh/department-of-veterans-affairs/va.gov-team/96186) (and [this](https://app.zenhub.com/workspaces/ivc-forms-652da2d3f0ae4c0016bfb109/issues/gh/department-of-veterans-affairs/va.gov-team/95333)), we are going to refine the Certifier Information flow of 10-10d to make it mandatory of all users. Then we're going to take the information entered there, and **if** the certifier is also "Applying for benefits for myself", we will populate the first applicant in the list with that information so they don't have to re-enter it. 

**Acceptance criteria**
- Certifier/signer role screen is removed
- Certifier/signer info flow is required of all users
- New checkbox for "I'm applying for benefits for myself" is added to the end screen of certifier info section
- When certifier is applying for benefits for himself, certifier information (name, address, phone, email) are automatically  duplicated to the first applicant slot in the list loop
- Certifier email field is required
- Form signature must always match the certifier name

**Comment by michaelclement** [comment]
Additional considerations the original ticket:
- Cypress test data will need to be updated
- Unit tests will need to be updated
- Adding the certifier to the applicants array should not damage any existing SIP data
- We will need a migration (or at least to change the version number of the form) since it has already technically been deployed in the production environment.

**Comment by michaelclement** [comment]
Current progress:
- [X] New functionality is in place
- [X] Unit tests have been updated
- [ ] Cypress tests have been updated
- [ ] corresponding vets-api changes

**Comment by michaelclement** [comment]
Running into some unanticipated difficulty with automatically prefilling an applicant's info if they are also the certifier. Going to have to take extra time to make sure I get it right, which is possibly going to prevent this from getting closed on Monday.

**Comment by michaelclement** [comment]
[PR with the bulk of the work is here](https://github.com/department-of-veterans-affairs/vets-website/pull/32857). See the **current progress** checklist above for what still needs to happen.

**Comment by michaelclement** [comment]
Work is done, testing completed. Just waiting on:
- [ ] Team approval
- [ ] Platform approval

**Comment by baltes** [comment]
Thanks @michaelclement. Can we add a flipper to this for review in staging? If not we can review on a call, but if possible I'd like to verify this one after a merge.

**Comment by michaelclement** [comment]
@baltes @jamiefiore @rspope This has been merged into main and should be visible on staging shortly. Please let me know if you have any additional feedback.
